packages/celotool/src/lib/blockscout.ts

```diff
@@ -215,3 +217,41 @@ export async function switchIngressService(celoEnv: string, ingressName: string)
    -p='[{"op": "replace", "path": "/spec/rules/0/http/paths/0/backend/serviceName", "value":"${ingressName}-web"}]'`
   await execCmdWithExitOnFailure(command)
 }
+
+export async function accessSecretVersion() {
```

Comment:
Wdyt about generalizing this function to provide the project, secret name and version as parameters as parameters and return the raw secret? Then the callers should be responsible of providing the parameters and parsing the data.